.github/workflows/todo_app_ci_pipeline.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           mongo_db_name: "${{secrets.MONGO_DB_NAME}}"
           git_client_id: "${{secrets.GIT_CLIENT_ID}}"
           git_client_secret: "${{secrets.GIT_CLIENT_SECRET}}"
+          LOGGLY_TOKEN: "${{secrets.LOGGLY_TOKEN}}"
         run: docker run -e mongo_db_connection -e mongo_db_name -e git_client_id -e git_client_secret my-test-image todo_app/e2e_tests
```

Comment:
It looks like you've added an environment variable here that you then don't use (since it's not passed into the `docker run ..` command); do we want the tests to log to Loggly? In some situations that might be useful, in others a waste of effort, but either way there's no point being in this halfway house!